src/openlifu/io/ustx.py

```diff
@@ -481,7 +482,7 @@ def get_pulse_data_registers(self, profile: Optional[int]=None, pack: bool=False
         levels = pattern['levels']
         lengths = pattern['lengths']
         nperiods = len(levels)
-        level_lut = {-1: 0b01, 0: 0b00, 1: 0b10}
+        level_lut = {-1: 0b01, 0: 0b11, 1: 0b10}  # Map levels to register values 0b11 drive to ground 0b00 high impedance
```

Comment:
@georgevigelette @peterhollender Please check this change introduced into `main` from `p/sonication_control_refactor`. Do we want this?